# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../silo/register-pseudoop.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./common/udivdi3.S.orig	2004-06-28 21:23:18.727522472 +0200
+++ ./common/udivdi3.S	2004-06-28 21:24:00.399187416 +0200
@@ -17,6 +17,8 @@
 the Free Software Foundation, 59 Temple Place - Suite 330,
 Boston, MA 02111-1307, USA.  */
 
+	.register %g2,#scratch
+
 	.text
 	.align 4
 	.globl __udivdi3
--- ./second/muldi3.S.orig	2004-06-28 21:29:22.416233384 +0200
+++ ./second/muldi3.S	2004-06-28 21:29:44.663851232 +0200
@@ -17,6 +17,8 @@
 the Free Software Foundation, 59 Temple Place - Suite 330,
 Boston, MA 02111-1307, USA.  */
 
+	.register %g2,#scratch
+
 	.text
 	.align 4
 	.globl __muldi3
--- ./common/divdi3.S.orig	2004-06-28 21:21:26.144637672 +0200
+++ ./common/divdi3.S	2004-06-28 21:21:52.257667888 +0200
@@ -17,6 +17,9 @@
 the Free Software Foundation, 59 Temple Place - Suite 330,
 Boston, MA 02111-1307, USA.  */
 
+	.register %g3,#scratch
+	.register %g2,#scratch
+ 
 	.data
 	.align 8
 	.globl	__clz_tab
